packages/azure-container-services/src/index.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const { isNodeJsTooOld, minimumNodeJsVersion } = require('@instana/core/src/util/nodeJsVersionCheck');
+
+// As of now, Azure App Service supports Node.js versions 16 and 18. However, for existing services, older
```

Comment:
TBD

>  az webapp list-runtimes --os linux | grep NODE
>  "NODE:20-lts",
>  "NODE:18-lts",
>  "NODE:16-lts",

Reply:
Thank you, node 20 support was not available when I wrote this code, so I corrected the line. Thanks:>